### PR TITLE
Improve performance of the parallax effect

### LINKED ISSUE #483: Homepage hero parallax performance issues
**Operating system, Browser, Browser Version:**
OS X 10.15.7
Browser: Chrome/108.0.0.0

Flickering and poor performance on homepage hero

https://user-images.githubusercontent.com/9118169/207429338-f31f2c9e-87f4-4111-8310-cdd7f752b5c5.mov

### PR DESCRIPTION
**Pull request by bits-by-brandon**
Fixes https://github.com/godotengine/godot-website/issues/483

https://user-images.githubusercontent.com/9118169/207452374-eb7a0137-6371-4abf-9d44-4df5d5c94353.mov

### CODE REVIEW

**Comment by YuriSizov** [comment]
I wonder if this would work well on mobile now, given that we hook into animation frames.

File: themes/godotengine/partials/footer.htm [hunk 1]
```diff
@@ -181,23 +181,27 @@ <h4 class="text-right"><a href="/contact">Contact us</a></h4>
...
                 const parallaxImage = document.querySelector('section.hero .background-image');
+                const parallaxMinWidth = 768;
+                const parallaxSpeed = 0.4;
 
-                window.addEventListener("scroll", () => {
-                  // Only scroll in desktop or tablet mode.
-                  if (window.innerWidth > parallaxWidth) {
-                    parallaxImage.style.backgroundPositionY = 'calc( 50% + ' + window.pageYOffset * 0.5 + 'px)';
+                const parallaxTick = (offsetY) => {
+                  console.log(offsetY);
```
Comment (YuriSizov) [inline themes/godotengine/partials/footer.htm]: Oops? 🙃

**Comment by bits-by-brandon** [comment]
Probably not, because mobile safari doesn't send scroll events every frame. I can do another implementation with a polling animation loop that should work better though

**Comment by bits-by-brandon** [comment]
Probably not, because mobile safari doesn't send scroll events every frame. I can do another implementation with a polling animation loop that should work better on mobile

**Review by coppolaemilio — APPROVED** [review]

**Pull request merged**